Source/Urho3D/Graphics/ShaderPrecache.cpp

```diff
@@ -119,7 +119,11 @@ void ShaderPrecache::LoadShaders(Graphics* graphics, Deserializer& source)
 
         // Check for illegal variations on OpenGL ES and skip them
 #ifdef GL_ES_VERSION_2_0
-        if (vsDefines.Contains("INSTANCED") || (psDefines.Contains("POINTLIGHT") && psDefines.Contains("SHADOW")))
+        if (
```

Comment:
Found this a bit clumsy, but couldn't figure a cleaner way without introducing annoying code repetition..